.github/workflows/pr.yaml

```diff
@@ -89,13 +90,18 @@ jobs:
     with:
       # Run checks for any PR branch
       run_check: ${{ fromJSON(needs.prepare.outputs.is_pr) }}
-      # Run conda-build for main/dev branches and PRs with the conda-build label
+      # Build morpheus-core conda package. This is done for main/dev branches and
+      # for PRs with the conda-core-build label
+      conda_core_run_build: ${{ !fromJSON(needs.prepare.outputs.is_pr) || fromJSON(needs.prepare.outputs.has_conda_core_build_label) }}
+      # Upload morpheus-core conda package only for non PR branches. Use 'main' for main branch and 'dev' for all other branches
+      conda_core_upload_label: ${{ !fromJSON(needs.prepare.outputs.is_pr) && (fromJSON(needs.prepare.outputs.is_main_branch) && 'main' || 'dev') || '' }}
+      # Run morpheus conda package, with all components. This is done for main/dev
+      # branches and for PRs with the conda-build label.
       conda_run_build: ${{ !fromJSON(needs.prepare.outputs.is_pr) || fromJSON(needs.prepare.outputs.has_conda_build_label) }}
-      # Update conda package only for non PR branches. Use 'main' for main branch and 'dev' for all other branches
       # Disable conda upload for now, once we have morpheus packages in conda forge set the value to
-      # !fromJSON(needs.prepare.outputs.is_pr) && (fromJSON(needs.prepare.outputs.is_main_branch) && 'main' || 'dev') || ''
       conda_upload_label: ""
```

Comment:
Shouldn't this be set to the same as `conda_core_upload_label`?

Reply:
Do we need to start uploading the morpheus package that includes all components? I created the separate upload label for the morpheus-core package mainly to hold this one back.

Reply:
Based on further discussion, we will not upload the "morpheus" conda package because of the pip dependencies.